admin/backend/adapters/mongoose/resource.js

```diff
@@ -110,6 +110,10 @@ class Resource extends BaseResource {
     }, {})
     return new ValidationError(`${this.name()} validation failed`, errors)
   }
+
+  databaseType() {
```

Comment:
FYI - here you don't have to write docs because its not an interface - its actual implementation